jpa/eclipselink/pom.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<parent>
+		<groupId>org.springframework.data.examples</groupId>
+		<artifactId>spring-data-jpa-examples</artifactId>
+		<version>1.0.0.BUILD-SNAPSHOT</version>
+	</parent>
+
+	<repositories>
```

Comment:
@olivergierke  , thanks again for the feedback! 

There isn't a non-snapshot version of the plugin available. As I was thinking more about this, I think the plugin from maven central and the use of a persistence.xml better illustrates the costs of using static weaving? Another approach would be to use the plugin from maven central, define a persistence.xml with only the entity information to support the static weaving during build time and then not use it for the Spring Boot application example. In this case, the persistence.xml exists just to provide the required information for the build time weaving of the entity, Spring Boot customizations are kept to a minimum and the infrastructure setup cost is all in the build. I just verified that works as well. 

The more detailed example would be using the maven central plugin and a persistence.xml and doing more Spring Boot customization to [use a traditional persistence.xm](http://docs.spring.io/spring-boot/docs/current/reference/html/howto-data-access.html#howto-use-traditional-persistence-xml).